network/load-balancer/concepts.mdx

```diff
@@ -44,7 +52,23 @@ For more information about balancing rules, refer to our [blog post "What is a l
 
 </Concept>
 
-<Concept opened>
+<Concept>
+
+## Certificate
+
+An certificate (aka SSL certificate / TLS certificate / TLS/SSL certificate) is a digital certificate that enables an encrypted connection between a client and a web server. 
+
+<Message type="tip">
+
+  You may hear certificates referred to as "SSL certificates", "TLS certificates" or "TLS/SSL certificates". These are all the same thing. SSL was the protocol initially used for encryption, though modern certificates in fact use TLS (Transport Layer Security) as the encryption protocol.
+
+</Message>
+
+You can [add a  certificate](/network/load-balancer/how-to/add-certificate) to your Load Balancer's frontend to enable the Load Balancer to encrypt/decrypt incoming traffic, necessary for [SSL bridging and SSL offloading configurations](/network/load-balancer/reference-content/configuring-load-balancer). Scaleway Load Balancer enables you to generate a Let's Encrypt certificate, or import a third-party or self-signed certificate.
```

Comment:
The load balancer is decrypting incoming client traffic and encrypting outgoing client traffic.

Reply:
corrected